### DISCUSSION TITLE
[Resolved] Multiple projects are stuck "Assigning Domains" nothing listed on Vercel status

### DISCUSSION THREAD

**Discussion by yoDon** [vercel/community#4594] [Bug Report]
### Page to Investigate

https://vercel.com/-team-name-/-project-name-/deployments

### Steps to Reproduce

Vercel has suddenly started taking 15-20 minutes to assign domains at the end of a deployment. 

This is happening on multiple deployments and multiple projects. No issues are listed on the Vercel Status page https://www.vercel-status.com

The Vercel deployments pages are also suddenly very slow to load

**Comment by jswent** [comment]
Same here, thought I was the only one having this problem. None of my projects have had their domains assigned. Still suck.

**Comment by isaiahscheel** [comment]


**Reply by jswent** [reply]
Yep, same for me but with Next 13. So don't think this is an adapter problem.

**Comment by davidlbowman** [comment]
Same here. Also, I cannot create a support case, because it doesn't recognize my professional account.

**Comment by leerob** [comment]
We're taking a look, thanks for letting us know 🙏

**Reply by amyegan** [reply]
We posted an incident report that you can follow for updates: https://www.vercel-status.com/incidents/ndnw1mxqjkh8

**Reply by amyegan** [reply]
The issue has been resolved. Please let us know if you continue to see problems with domains 🙏

**Comment by danblock97** [comment]
Glad to see it's not just me, came here to raise this as an issue. I'll hold off on carrying on trying to redeploy.

**Comment by tjelz** [comment]
Can also confirm I am having issues

**Comment by shanejones** [comment]
Also confirming this issue. Tried multiple times to restart and didn't see anything on the status page.

**Comment by Chris-Barnard** [comment]
Glad to see I'm not the only one with an issue :)  I'll keep monitoring here for an update

**Comment by mukundagujjar** [comment]
Same issue. It worked fine in my local environment and was worried if I missed some additional configuration while pushing to production.

**Comment by ilidazilv** [comment]
Same issue

**Comment by yannJaster10i9** [comment]
Same issue here !
